app/views/organizations/_details.html.erb

```diff
@@ -80,7 +80,7 @@
                 <h3 class='font-bold'>Default Intake Location</h3>
                 <p>
                   <%= fa_icon "building" %>
-                  <%= StorageLocation.find_by(id: @organization.intake_location)&.name || "Not defined" %>
```

Comment:
This switches intake location to default location... I think that's wrong?

Reply:
Thanks, totally missed that somehow. Fixed in last commit [44f12db](https://github.com/rubyforgood/human-essentials/pull/5043/commits/44f12dbe5f0c94314d4606d5cbef7574e3f773bb)